include/deal.II/base/mpi.h

```diff
@@ -1195,6 +1203,28 @@ namespace Utilities
               const T &          object_to_send,
               const unsigned int root_process = 0);
 
+    /**
+     * Broadcast the information in @p buffer from @p root to all
+     * other ranks.
+     *
+     * Like `MPI_Bcast` but with support to send data with a @p count
+     * bigger than 2^31. The datatype to send needs to be supported
+     * directly by MPI and is automatically deduced from T.
+     *
+     * Throws an exception if any MPI command fails.
+     *
+     * @param buffer Buffer of @p count objects
+     * @param count The number of objects to send
+     * @param root The rank of the process with the data
+     * @param comm The MPI communicator to use
```

Comment:
Can you document explicitly that both the sender and the receiver have to know the `count` here?

Also, put periods at the ends of each of these lines.